⚠️ Looks like there is a build failure on iOS due to these changes:

```
gutenberg/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift:472:37: error: protocol 'Encodable' as a type cannot conform to the protocol itself
            let metadataData = try? JSONEncoder().encode(self.metadata),
                                    ^
gutenberg/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift:472:37: note: only concrete types such as structs, enums and classes can conform to protocols
            let metadataData = try? JSONEncoder().encode(self.metadata),
                                    ^
gutenberg/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift:472:37: note: required by instance method 'encode' where 'T' = 'Encodable'
            let metadataData = try? JSONEncoder().encode(self.metadata),
```

It's interesting because I can't reproduce it when using Xcode 14+. In fact, I only get it when using Xcode 13 🤔. I'll check this deeper.

**UPDATE:** This is actually caused by the Swift version:
- Swift version `5.6` (Xcode 13): Fails ❌ 
- Swift version `5.7` (Xcode 14) Succeeds ✅ 

Specifically, seems related to a new feature introduced in `5.7`: [**_Implicitly Opened Existentials_**](https://github.com/apple/swift-evolution/blob/main/proposals/0352-implicit-open-existentials.md)

--- a/packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift
+++ b/packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift
@@ -5,20 +5,20 @@ public struct MediaInfo: Encodable {
     public let title: String?
     public let caption: String?
     public let alt: String?
-    public let metadata: Encodable
+    public let metadata: [String: Any]
 
     private enum CodingKeys: String, CodingKey {
         case id, url, type, title, caption, alt
     }
 
-    public init(id: Int32?, url: String?, type: String?, caption: String? = nil, title: String? = nil, alt: String? = nil, metadata: Encodable? = nil) {
+    public init(id: Int32?, url: String?, type: String?, caption: String? = nil, title: String? = nil, alt: String? = nil, metadata: [String: Any] = [:]) {
         self.id = id
         self.url = url
         self.type = type
         self.caption = caption
         self.title = title
         self.alt = alt
-        self.metadata = metadata ?? [:] as [String: String]
+        self.metadata = metadata
     }
 }
 

--- a/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift
+++ b/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift
@@ -468,16 +468,13 @@ extension MediaInfo {
     func encodeForJS() -> [String: Any] {
         guard
             let data = try? JSONEncoder().encode(self),
-            var jsonObject = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any],
-            let metadataData = try? JSONEncoder().encode(self.metadata),
-            let metadataJsonObject = try? JSONSerialization.jsonObject(with: metadataData, options: []) as? [String: Any]
-        else
+            var jsonObject = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: Any] else
         {
             assertionFailure("Encoding of MediaInfo failed")
             return [String: Any]()
         }
 
-        jsonObject["metadata"] = metadataJsonObject
+        jsonObject["metadata"] = self.metadata
         return jsonObject
     }
 }

